erlang/kivra_api_errors.app.src

```diff
@@ -0,0 +1,14 @@
+{application, kivra_api_errors,
+ [{description, "Kivra API Errors and Status Codes"},
+  {vsn, git},
+  {registered, []},
+  {applications,
+   [kernel,
+    stdlib,
+    jsx
+   ]},
+  {env,[]},
+  {modules, []},
+  {licenses, ["MIT"]},
```

Comment:
Do we have some internal guidelines for which license to use?

Reply:
I think we use MIT in most places